lib/components/user/form-navigation-buttons.tsx

```diff
@@ -15,19 +15,24 @@ const RightButton = styled(Button)`
   float: right;
 `
 
-interface ButtonType {
+export interface ButtonType {
+  /** The component to use for the button. */
+  ButtonComponent?: ReactNode
+  /** Whether the button is disabled. */
   disabled?: boolean
   /** Triggered when the button is clicked. */
   onClick?: MouseEventHandler<Button>
   /** The text to display on the button (JSX elements accepted). */
-  text: ReactElement
+  text?: ReactElement
```

Comment:
Why is this optional?

Reply:
this button text was already defined in our separate `DeleteForm` component with i18n, so we didn't need it for this button and then we decided to make it optional

Reply:
Hm I'm not sure I love that... Can we name it `textOverride` or something?